http/http.go

```diff
@@ -19,34 +22,68 @@ type Server struct {
 
 // Listen starts the http server on the given port.
 func (server *Server) Listen(port int) {
-	r := mux.NewRouter()
-	s := r.PathPrefix(indexPath)
-	s.HandlerFunc(createIndex(server.Index)).Methods(http.MethodPost)
+	router := mux.NewRouter()
+	router.HandleFunc("/{index}/{id}", createIndex(server.Index)).Methods(http.MethodPost)
```

Comment:
We should discuss if we want to be as similar as possible to elastic search. If yes, the elastic search API doesn't use an api nor index prefix in their routes, if no, I agree with you, we should prefix the routes with `/api`.